### ISSUE TITLE
Non working days not visible when zooming out

### ISSUE THREAD

**Issue by matsbryntse** [bryntum/support#2788]
[Forum post](https://www.bryntum.com/forum/viewtopic.php?f=44&t=17360&p=86124#p86124)

I created calendar for resources, like on that screen:
![zoom1.png](https://www.bryntum.com/forum/download/file.php?id=13126)
(I edited background in css)

But after zoomout, calendar is not visible. It's looking like viewPresets are not supporting this.
![zoom2.png](https://www.bryntum.com/forum/download/file.php?id=13127)

How to set calendar always visible? Or for specified viewPreset?
It's perfect working with resourceTimeRange - view preset is not recognized and ranges are always visible.

Greetings.
![zoom1.png](https://www.bryntum.com/forum/download/file.php?id=13126)
![zoom2.png](https://www.bryntum.com/forum/download/file.php?id=13127)

**Comment by matsbryntse** [comment]
Related: https://github.com/bryntum/support/issues/2798

**Comment by arcady-zherdev** [comment]
Related request to disable on zoom magic hiding intervals: https://www.bryntum.com/forum/viewtopic.php?p=88961#p88961

**Comment by arcady-zherdev** [comment]
We ended up adding a new config to `nonWorkingTime` feature.
```
* Added a new `hideRangesOnZooming` config to `NonWorkingTime` feature (Fixed #2788).
  The config allows to disable the feature default behavior when it hides ranges
  shorter than the base timeaxis unit on zooming out.
```

So to get the behavior you want you will need to set it to `false`:
```js
        scheduler = new Scheduler({
            ...
            features   : {
                nonWorkingTime : {
                    hideRangesOnZooming : false
                }
            }
        });
```
